pages/learn/manage/device-statuses.md

```diff
@@ -11,20 +11,21 @@ The application overview page shows the status of each device in the application
-|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Online            | The device is online and is connected to the VPN or has recent API communications.                                                                       |
-| Configuring       | The device is applying OS configuration.                                                                                                                 |
-| Updating          | The device is updating to a new application release.                                                                                                     |
-| Offline           | The device is offline and is not connected to VPN and has not any recent API communications.                                                             |
-| Post Provisioning | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                      |
-| Inactive          | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API. |
+| Status            | Description                                                                                                                                                     |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to the VPN or has recent API communications.                                                                              |
+| Configuring       | The device is applying OS configuration.                                                                                                                        |
+| Updating          | The device is updating to a new application release.                                                                                                            |
+| Offline           | The device is offline and is not connected to VPN and has not any recent API communications.                                                                    |
+| Post Provisioning | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                             |
+| Inactive          | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.        |
+| Frozen            | The device has been frozen because it's outside the organization's allowance, or is in a paid [application type][application type] on a free tier organization. |
```

Comment:
It would be great to link to a pagee that explains more about "frozen" devices, if one exists.

Reply:
Seems that atm we don't have such a page about device freezing. We definitely should have one, but it's also weird what it should say, given the nature of the current implementation.